analysers/trials/processDuration.py

```diff
@@ -29,16 +30,50 @@
 
 # TODO: Use Spark for all computations
 
+dataRDD = sc.cassandraTable(cassandraKeyspace, srcTable)\
+        .select("process_definition_id", "source_process_instance_id", "end_time", "start_time", "duration") \
+        .where("trial_id=? AND experiment_id=?", trialID, experimentID) \
+        .filter(lambda r: r["process_definition_id"] is not None) \
+        .cache()
+
+processes = dataRDD.map(lambda r: r["process_definition_id"]) \
```

Comment:
@VincenzoFerme This one I'm manipulating the RDD differently to get a simple list of process durations, so I have to do it like this with Spark.